acquia/blt
# Update drush to 9.6.0, and robo to 1.3.4

Fixes #3350 
--------

Changes proposed:
---------
Update drush to 9.6.0. Requires Robo updated to 1.3.x.

Caused by https://github.com/consolidation/Robo/commit/3317048f90b78101619d674514101cd99d9ca0a6#diff-20b8d9734807c101cc969094af11a517.

Steps to replicate the issue:
----------
(If this PR is not fixing a defect/bug, you can remove this section.)

1. Update to drush 9.6.0
2. Run blt list
3. See that it fails
Steps to verify the solution:
-----------
(How does someone verify that this does what you think does?)

It's covered by tests. A manual test running `blt list` on this branch would be enough.
```
git clone https://github.com/alexxed/blt.git
rm -rf blted8
cd blt
git checkout 3350
composer update
./vendor/bin/robo create:from-symlink
cd ../blted8
blt list
